src/views/dataaccesscoe/pl-planning.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div class="usa-layout-docs__main desktop:grid-col-9 usa-prose">
+    <div class="dh-data-access-coe_project-lifecycle_subpage_title">
+      <DOTProjectLifecycleImg height="170" width="170" active="planning"/>
+      <h1>Planning</h1>
+    </div>
+    <p>
+      The Planning project lifecycle phase occurs when planning a project or procurement begins through release of a project or procurement. During this phase, Program/Project Managers work on planning and releasing the project or procurement and Data Providers start brainstorming project ideas. See the appropriate page for Program/Project Manager and Data Provider resources.
+    </p>
+
+    <!-- Tab links -->
+    <div class="dh-data-access-coe_tab">
+      <button :class="active_tab=='id_tab1'?'dh-data-access-coe_tab-active':''" @click="tabClicked($event, 'id_tab1')">Program/Project Manager</button>
+      <button :class="active_tab=='id_tab2'?'dh-data-access-coe_tab-active':''" @click="tabClicked($event, 'id_tab2')">Data Providers</button>
+    </div>
+
+    <!-- Tab content -->
+    <div v-if="active_tab=='id_tab1'" id="id_tab1" class="usa-layout-docs__main desktop:grid-col-10 dh-data-access-coe_tab-content">
+      <p>
+        To ensure the Data Access CoE provides the maximum benefit to a project or procurement, we recommend engaging the Data Access CoE team in at the beginning of the project lifecycle. To begin this engagement during the Planning phase, you can find resources on this page and contact us directly at
+        <a href="mailto:data.itsjpo@dot.gov">data.itsjpo@dot.gov<img class="in-line-external-link-icon" src="/images/icons/external-link_1b1b1b.svg" alt="External from DOT link icon" title="External link"></a>.
+      </p>
+      <p>
+        During Planning, the Data Access CoE can support Program/Project Managers by:
+      </p>
+      <ul>
+        <li>
+          Advising on data access requirements clauses and solicitation language for Notice of Funding Opportunities (NOFOs), Requests for Proposals (RFPs) and internal ITS JPO-sponsored projects. We have standard language and clauses that can be tailored to your needs.
```

Comment:
`We` should be replaced with a neutral third person

Reply:
tech-writer review is required

Reply:
Approved and changes made